generate-dag

```diff
@@ -78,14 +78,13 @@ if __name__ == '__main__':
         vmu.die('%s: parameter directory "%s" does not exist' % (script_name, param_dir))
 
     run_params = vmu.load_run_params(param_dir)
-    vmu.pkg_mapping(run_params) # verify uniqueness of package sets + labels
-    
+    vmu.flatten_run_params(run_params) # verify uniqueness of package sets + labels
+
     # Set up test run directory
     test_run_directory = os.getcwd()
 
     # Check if nightly
     with open('run_label', 'r') as f:
-        global NIGHTLY
         NIGHTLY = f.read().strip() == 'nightly'
```

Comment:
Very minor detail, but `NIGHTLY` being in all-caps usually designates a global variable.  Now that it's just a function-local variable, do we want to change the variable name to something not all-caps?